From 605dd6a4ea94277044fd483e9ecb567dd3c355fe Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Tue, 8 Jun 2021 21:58:57 -0500
Subject: [PATCH 02/11] CMake: set CMP0072 to NEW

Silences CMake's warning and uses GLVND, which still works for linux

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
Change-Id: Iadd173b41c4fbcdff9fe512f9a7dbf70a6d95bcd
---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 13c45ec8..bd96f596 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,6 +8,10 @@
 
 cmake_minimum_required(VERSION 3.5)
 
+if(POLICY CMP0072)
+  cmake_policy(SET CMP0072 NEW)
+endif()
+
 project(WebP C)
 
 # Options for coder / decoder executables.
-- 
2.31.1

